generators/app/templates/package.json

```diff
@@ -4,7 +4,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.8.0",
     "@fortawesome/free-solid-svg-icons": "^5.6.3",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@inrupt/solid-react-components": "^0.4.4-rc.0",
+    "@inrupt/solid-react-components": "^0.4.3-local.14",
```

Comment:
Don't check in local version numbers as this will corrupt the develop branch

Reply:
done